src/components/action-popover/action-popover-menu/action-popover-menu.component.js

```diff
@@ -164,6 +166,8 @@ ActionPopoverMenu.propTypes = {
   onClick: PropTypes.func,
   /** @ignore @private */
   placement: PropTypes.oneOf(["bottom", "top"]),
+  /** Horizontal alignment of menu items content */
+  horizontalAlignment: PropTypes.oneOf(["left", "right"]),
```

Comment:
Can you update `d.ts` file please?

Reply:
it's an internal prop, passed from the parent component